source/appModules/mpc-hc64.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: UTF-8 -*-
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2018 NV Access Limited, Łukasz Golonka
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+from mplayerc import *
```

Comment:
Could you please add an empty line before and after this line?

Reply:
@leonardder Done